02-express-tutorial/app.js

```diff
@@ -3,22 +3,40 @@
 const express = require('express');
 const { products } = require('./data');
 const app = express();
+const people = require('./data.js')
+const peopleRouter = require('./routes/people');
 
-app.use(express.static("./public"));
 
-app.get('/api/v1/test', (req, res) => {
+const logger = (req, res, next) => {
+    const currentTime = new Date().toISOString();
+    console.log(`[${currentTime}] ${req.method} ${req.url}`);
+    next();
+};
+
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+
+app.use(logger);
```

Comment:
This line adds the `logger` middleware to all paths, so you don't need to add it to individual paths below. In fact, adding it twice causes it to double-print the log messages, which could be confusing!